core/dbt/cli/requires.py

```diff
@@ -70,7 +71,7 @@ def wrapper(*args, **kwargs):
                 CommandCompleted(
                     command=ctx.command_path,
                     success=success,
-                    completed_at=get_json_string_utcnow(),
+                    completed_at=Timestamp().GetCurrentTime(),
```

Comment:
Looks like Gerda preferred the approach with `get_json_string_utcnow()` in https://github.com/dbt-labs/dbt-core/pull/7217, I'm guessing to avoid sprinkling `google.protobuf` types around the codebase / outside the `events` module